packages/web-runtime/src/components/UserMenu.vue

```diff
@@ -88,7 +88,7 @@
 </template>
 
 <script>
-import appVersionJson from '../../build/version.json'
+import { version } from '../../package.json'
```

Comment:
Nice, but version in that package.json is not up to date. Also, would be quite nice to have only one version defined in the top level package.json (has no version at all at the moment).